lib/commands/audit.js

```diff
@@ -1,8 +1,10 @@
 const Arborist = require('@npmcli/arborist')
 const auditReport = require('npm-audit-report')
-const reifyFinish = require('../utils/reify-finish.js')
-const auditError = require('../utils/audit-error.js')
+
 const ArboristWorkspaceCmd = require('../arborist-cmd.js')
+const auditError = require('../utils/audit-error.js')
+const reifyFinish = require('../utils/reify-finish.js')
+const VerifySignatures = require('../utils/verify-signatures.js')
```

Comment:
We're either going to have to move this code into this file, or update the coverage map.  I'd prefer the former, I think the code footprint won't be too onerous once we remove the cacache stuff and iterate a few times on it.